### DISCUSSION TITLE
When drawing a curve, shouldn't the line follow the pencil's lead?

### DISCUSSION THREAD

**Discussion by irenevera** [uxbox/uxbox#579] [Ideas]
![penpot](https://user-images.githubusercontent.com/72253924/107036506-0cb99900-67ba-11eb-8bbb-14854fe30f25.gif)

That was a bit of an excuse to say hello👋 and that I'm in love with how smooooooooth this whole thing works!! 🧼🧼🧼🧼

**Comment by myfunnyandy** [comment]
That's the spirit!

About the curve tool, we know that is in a very rought shape. We have planned to do a total rework of it.
